site/content/docs/latest/tutorials/kubeapps-on-tkg/step-3.md

```diff
@@ -5,60 +5,65 @@ Once Kubeapps is up and running on the VMware Tanzu™ Kubernetes Grid™ cluste
 The general procedure to add any repository is described below:
 
 1. Log in to Kubeapps as described in the previous step.
-2. Click the menu button in the top right corner (dotted square).
+2. Click the **right menu\*** button in the top right corner (dotted square).
 
    ![Kubeapps home](../../img/kubeapps-on-tkg/kubeapps-applications-empty.png)
 
-3. Click the _Package Repositories_ button.
+3. Click the **Package Repositories** option in the right menu to navigate to the `Package Repositories` page.
 
    ![Right menu](../../img/kubeapps-on-tkg/kubeapps-menu-right.png)
 
-4. Add a new repository by clicking the _Add Package Repository_ button.
+4. Add a new repository by clicking the **Add Package Repository** button.
 
-> **NOTE**: If no repositories were specified in the `initialRepos` chart parameter during the Kubeapps installation, no repositories will be shown on the _Application Repositories_ page.
+   > **NOTE**: If no repositories were specified in the `initialRepos` chart parameter during the Kubeapps installation, no repositories are displayed on the `Package Repositories` page.
 
 5. Fill the _Add a Package Repository_ form using the guidelines below:
 
-- The _Name_ field is a friendly display name for the repository.
-- The _URL_ field specifies the endpoint of the repository. This endpoint might require different forms of authentication, such as `None`, `Basic Auth` (username and password), `Bearer Token` (a token) or another `Custom` mechanism.
+- `Name` field is a friendly display name for the repository.
+- `URL` field specifies the endpoint of the repository. This endpoint might require different forms of authentication, such as `None`, `Basic Auth` (username and password), `Bearer Token` (a token) or another `Custom` mechanism.
+- `Description` field (optional) is a brief summary for the repository.
+- `Packaging format` field specifies the format of the packages available in the repository.
+- `Scope` field specifies where the packages from the repository could be installed in the cluster (globally on any namespace or limited to an specific namespace in the cluster).
+- `Package Storage Type` field specifies the type of repository. Currently, Kubeapps supports both Helm repositories and OCI registries.
```

Comment:
Just FTR, this field is also applicable to Carvel, I think. We are using imgPkgBundle, but others are also available.